packages/gateway/src/features/metrics/schema.graphql

```diff
@@ -153,4 +167,7 @@ type Query {
     locationId: String!
     count: Int!
   ): TimeLoggedMetricsResultSet
+  getUserAuditLog(
+     practitionerId: String!
+   ): UserLoggedMetricsResultSet
```

Comment:
`UserLoggedMetricsResultSet` could be named as `UserAuditResultSet`. Let's call `UserLogMetrics` a `UserAuditItem`